fstar-helpers/core-models/Cargo.toml

```diff
@@ -12,6 +12,7 @@ publish = false
 [dependencies]
 rand = "0.9"
 hax-lib.workspace = true
+pastey = "0.1.0"
```

Comment:
What is this dependency needed for?

Reply:
This is for https://github.com/cryspen-ext/core-models/commit/ebdb0b68e3002dca6df891b858fccc3bbea6c05d.

We killed From instances, and use explicit `from_*` and `to_*`.
This requires crafting Rust names, and this is not possible in stable Rust (https://doc.rust-lang.org/std/macro.concat_idents.html).